hw-passthrough/serial/serial-over-network.md

```diff
@@ -0,0 +1,220 @@
+# Connecting to serial port over network
+
```

Comment:
Johnson, can you move this file to /samples/serial/serial-over-network.md

Reply:
moved to /samples/serial/serial-over-network.md